modules/unsupported/vsi/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0                                 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.geotools</groupId>
+    <artifactId>unsupported</artifactId>
+    <version>27-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.geotools</groupId>
+  <artifactId>gt-vsi</artifactId>
+  <packaging>jar</packaging>
+  <name>VSI Plugin</name>
+
+  <description>Datastore plugin for VSI Virtual File System support.</description>
+
+  <properties>
+    <gdal.version>3.2.0</gdal.version>
```

Comment:
It would be best if all the GDAL based plugins use the same version of GDAL, otherwise it might be be impossible to mix imageio-ext GDAL, the OGR store, and the VSI store in the same application.
Right now imagieo-ext and the gt-ogr-jni modules depend on GDAL 2.2.0

See also https://github.com/geosolutions-it/imageio-ext/pull/253

Reply:
Thanks for outlining this - I've changed it to be in line with the other store types. It's worth noting this comes at the expense of some additional GDAL options not being available but can be re-enabled by advanced users by changing the bindings to a newer version manually.